### PR TITLE
[com_fields] Fix language strings/unknown columns/sorting

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Fix language strings and columns.
Fix Field Group header not sorting.

### Testing Instructions
Go to front end.
Edit an article.
Click on the Field button.
Right click on the modal to bring up a contextual menu to open `This Frame > Open Frame in New Tab`.
View the dropdown in the upper right corner.
Click on the Field Group header. 

Go to back end.
Go to Content > Fields
Make sure there are fields assigned to field groups.
Click on Field Group header.
Click on Language header.

### CODE REVIEW

**Comment by richard67** [comment]
I don't understand test step "Open the frame in a new tab.". You mean the modal for selecting the field? How shall I do that?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by Quy** [comment]
Updated testing instructions. Thanks.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 1f7caf0c51474ffe994f475e1243c2e82064adf3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by SharkyKZ** [comment]
Can you also change `category_title` to `g.title` to fix this:

>Column not found: 1054 Unknown column 'category_title' in 'order clause'

**Comment by infograf768** [comment]
> Can you also change category_title to g.title to fix this:

Looks like we also need to take off `category_title` from here
https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_fields/models/fields.php#L47

File: administrator/components/com_fields/models/fields.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function __construct($config = array())
 				'checked_out_time', 'a.checked_out_time',
 				'created_time', 'a.created_time',
 				'created_user_id', 'a.created_user_id',
-				'category_title',
+				'category_title', 'g.title',
```
Comment (SharkyKZ) [inline administrator/components/com_fields/models/fields.php]:
```suggestion
				'group_title', 'g.title',
```

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on 4b641aa80d47b012d1da953f4b98bee574bca00d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by infograf768** [comment]
Found another incongruity in backend Field modal when opening frame in new window. See `Language` at bottom of dropdown.

Screen Shot 2019-07-10 at 09 17 35

If I modify
https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_fields/views/fields/view.html.php#L207 to use `a.language`
and then in
https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_fields/views/fields/tmpl/modal.php#L60
same change.
I do get correctly
Screen Shot 2019-07-10 at 09 45 44

Let's solve also in this PR?

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on e75de2f9ba5ce31a69e7527e80e7da27a5837885<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by richard67** [comment]
In the frontext test I see untranslated language strings in the pull down.
I don't think it is related to this PR, but have you both @Quy and @infograf768 applied any other fix which I haven't applied yet? I've fetched clean staging just right now and made a fresh, clean install, and patch I applied with patchtester.

**Comment by Quy** [comment]
Yes, you also have to install #25475 to fix the untranslated language strings which is not related to this PR.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e75de2f9ba5ce31a69e7527e80e7da27a5837885<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25476">issues.joomla.org/tracker/joomla-cms/25476</a>.</sub>

**Comment by infograf768** [comment]
Restarted drone

**Comment by infograf768** [comment]
tks

**Pull request merged**